TooltipDecorator/tests/TooltipDecorator-specs.js

```diff
@@ -0,0 +1,16 @@
+import '@testing-library/jest-dom';
+import {render, screen} from '@testing-library/react';
+
+import TooltipLabel from '../TooltipLabel';
+
+describe('TooltipDecorator', () => {
+	describe('TooltipLabel', () => {
+		test('should render a tooltip label', () => {
+			render(<TooltipLabel>Label</TooltipLabel>);
+
+			const label = screen.getByText('Label');
+
+			expect(label).toBeInTheDocument();
+		});
+	});
+});
```

Comment:
Please add a few more tests (refering to tooltipPosition, for example. check out agate tests).